src/main/java/com/bhoruka/bloodbank/model/response/SearchCampResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.bhoruka.bloodbank.model.response;
+
+import java.util.Date;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class SearchCampResponse {
+
+    String id;
+
+    String partnerId;
+
+    Date dateOfCamp;
+
+    Long expectedNoOfDonors;
+
+    Long actualNoOfDonors;
+
+    Integer pageNo;
```

Comment:
Instead of returning the page number, is there a way we can return the total number of results instead? That way calculating page number will be easy as we can just do totalResults/pageSize. This will also allows us to change the pageSize and still use the same request.

Reply:
We can return the number of rows fetched from the database and return that as totalNoOfResponse to get how many camps are present, which can be used to calculate and modify pageSize. Does that work?

Reply:
What do you mean by 'the number of rows fetched from the database'? The database query will only return rows equal to the size of the page being passed since we want a paginated query, and if you return that number, our calculation will always show that only a single page exists, which might not be correct.
However, if you mean this is the total number of rows which match the given query then, yes, that works.

Reply:
Yes yes... Total no. of rows 🥇